### DISCUSSION TITLE
Re-building Image for Secondary Boards

### DISCUSSION THREAD

**Discussion by TrivialPants** [WLAN-Pi/feedback#8] [General Inquiries] [answered]
Hi All,

I wanted to test building the wlanpi image on another board which I believe may work okay (please correct if you think I may be wrong). I am thinking of using the OrangePI Zero 2 board, and I was hoping to take a list of requirements or a base image and just use the armbian build tool and build the image for that board. Is that a possibility? Or will I need to just install the Armbian image for the orangepi and install the software (Iperf, eperf, and so on) after I have the image working?

Thanks for any input!

-Taylor

**Comment by jolla** [comment]
The main requirement for the WLAN Pi project that is missing from the OrangePi Zero2 is the lack of OTG, but if that isn't required for your use case it should be possible to build a WLAN Pi image for it. You can find the build repo here: https://github.com/WLAN-Pi/build

**Comment by TrivialPants** [comment]
Awesome! Thank you for the quick response.

**Comment by danielmundi** [comment, marked as answer]
@TrivialPants, after cloning the build repository, you can use this command to generate the build:

```
sudo ./compile.sh BOARD=orangepizero2
```

Just a heads up - Armbian support for Orange Pi Zero 2 is still Work In Progress, so you might find some bugs :)

Let us know how this went!

**Reply by TrivialPants** [reply]
Hey just wanted to provide an update - let me know if I need to move this to another area. I am getting this response when I try to build with the above command. 

[ error ] ERROR in function source [ main.sh:246 ]
[ error ] Board configuration does not define valid kernel config 
[ o.k. ] Process terminated 

Do I need to provide a new conf file for the orangepizero2?

**Reply by TrivialPants** [reply]
Just wanted to provide some updates. I am having trouble similar to some of these folks in this armbian forum thread https://forum.armbian.com/topic/16170-orangepi-zero2-allwinner-h616/page/3/. This board may still be too early even if we get it to build the image for wlanpi. Just wanted to share that & thanks for any help you provided!

**Reply by danielmundi** [reply]
Thanks for the feedback.

Actually this board was included in Armbian after we shipped the 2.0.1 image, so the config for it is only on our `dev` branch, not on `main`.

Please try to `git checkout dev` first and then build with above command.
